platforms/quorum/configuration/roles/setup/geth/tasks/main.yaml

```diff
@@ -4,46 +4,46 @@
 #  SPDX-License-Identifier: Apache-2.0
 ##############################################################################################
 
+# This task registers a temporary directory
+- name: Register temporary directory
+  tempfile:
+    state: directory
+  register: tmp_directory
+ 
 # This task checks if the geth binary is already in place or not
 - name: Check geth
   stat:
     path: "{{ bin_install_dir }}/geth"
   register: geth_stat_result
 
-  # This task checks if quorum repo directory exists
-- name: Check quorum repo dir exists
-  stat:
-    path: "{{ bin_install_dir }}/quorum"
-  register: quorum_stat_result
-
-# This task clones the quorum git repository
-- name: Clone the git repo
-  git:
-    repo: "{{ geth.repo }}"
-    version: master
-    force: yes
-    dest: "{{ bin_install_dir }}/quorum"
-  when: not ( quorum_stat_result.stat.exists and geth_stat_result.stat.exists )
-
-# This task builds the geth binary
-- name: Make geth
-  make:
-    chdir: "{{ bin_install_dir }}/quorum"
-    target: geth
+# This task downloads the geth tar
+- name: Download geth tar 
+  get_url:
+    url: "{{ get_url.url }}"
+    dest: "{{ tmp_directory.path }}"
+    mode: 0440
   when: geth_stat_result.stat.exists == False
 
-# This task creates the bin directory, if it doesn't exist, for storing the geth binary
+# This task creates the bin directory, if it doesn't exist
 - name: Create bin directory
   file:
     path: "{{ bin_install_dir }}"
     state: directory
   when: geth_stat_result.stat.exists == False
 
-# This task puts the geth binary to above created bin directory
+# Extract the geth tarball 
+- name: Extract the geth tarball
+  unarchive:
+    src: "{{ tmp_directory.path }}/geth-alltools-linux-amd64-1.10.0-56dec25a.tar.gz"
+    dest: "{{ bin_install_dir }}"
```

Comment:
If you have created a temporary directory to download the geth tar file, unzip the tar in the temporary directory and then pick the binary from that temporary directory to the `bin_install_dir` var